dev/insights/galaxy_ng.env

```diff
@@ -1,6 +1,7 @@
 PULP_CONTENT_PATH_PREFIX=/api/automation-hub/v3/artifacts/collections/
 
 PULP_GALAXY_API_PATH_PREFIX=/api/automation-hub/
+PULP_API_ROOT=/api/automation-hub/pulp/
```

Comment:
could we find a way to configure this a little more easily? Maybe put it in dynaconf hooks so that it doesn't have to be configured in so many different places? Something like this could work:

``` python
# /galaxy_ng/galaxy_ng/app/dynaconf_hooks.py
def configure_api_base_path(settings: Dynaconf) -> Dict[str, Any]:
    galaxy_base = settings.get("GALAXY_API_PATH_PREFIX", None)

    if galaxy_base:
        return {
            "API_ROOT": f"/{galaxy_base.strip('/')}/pulp/"
        }
    else:
        return {}
```

This won't allow anyone to set `API_ROOT` by itself, but perhaps we could add some logic to only alter the value if it's set to a default value. @rochacbruno do you have any thoughts?

Reply:
@awcrosby one strategy you could use is create a new setting called something like `CORE_API_ROOT` that lets users set `API_ROOT`. Something like:

```
if core_api_root:
    API_ROOT=core_api_root
else:
    API_ROOT=galaxy_base + "/pulp/"
```

Reply:
thanks, i will give this a try

Reply:
@newswangerd PTAL - I went with the more simple approach of not having the pulp API_ROOT be user-configurable in our settings.py